pkg/prometheus/client.go

```diff
@@ -0,0 +1,66 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/api"
```

Comment:
sort imports as we have in the rest of the code base - standard imports first, third party imports second, our imports third.

Reply:
+1